frontend/src/metabase-lib/binning.ts

```diff
@@ -18,22 +18,22 @@ export function isBinnable(
   query: Query,
   stageIndex: number,
   column: ColumnMetadata,
-) {
+): boolean {
   return availableBinningStrategies(query, stageIndex, column).length > 0;
 }
 
 export function withBinning(
   column: ColumnMetadata,
   binningStrategy: Bucket | null,
-) {
+): ColumnMetadata {
   return ML.with_binning(column, binningStrategy);
 }
 
 export function withDefaultBinning(
   query: Query,
   stageIndex: number,
   column: ColumnMetadata,
-) {
+): ColumnMetadata {
```

Comment:
At a first glance these functions looked to me like they return `Query`, and I lost some time debugging this, so I defined the return types explicitly.